python-spec/src/somacore/query/query.py

```diff
@@ -298,7 +299,7 @@ def to_anndata(
 
```

Comment:
Let's add a description of what `drop_levels` does ...

Reply:
@pablo-gar @eddelbuettel @mojaveazure are you okay with `drop_levels`?

Reply:
(Presuming we'll want a same-ish name in Python & R both ...)

Reply:
Seurat uses `drop` to match R, but I'm fine with `drop_levels` in SOMA

Reply:
Sounds good to me!